Makefile.sdk.mk

```diff
@@ -157,6 +159,22 @@ build/fs-%/manifest/libdwarf.pkg: build/fs-env-%.rc build/fs-tmp-%/libdwarf/Make
 		done \
 		&& install -d $$prefix/lib \
 		&& install -m 644 $$builddir/libdwarf/.libs/libdwarf.a $$prefix/lib \
+		&& install -d $$prefix/lib/pkgconfig \
+		&& ( \
+			echo "prefix=\$${frida_sdk_prefix}"; \
+			echo "libdir=\$${prefix}/lib"; \
+			echo "includedir=\$${prefix}/include"; \
+			echo ""; \
+			echo "Name: libdw"; \
+			echo "Description: elfutils library for DWARF data and ELF file or process inspection"; \
+			echo "Version: 0.182"; \
+			echo "URL: http://elfutils.org/"; \
+			echo "Libs: -L\$${libdir} -ldw"; \
+			echo "Cflags: -I\$${includedir}"; \
+			echo "Requires: libelf = 0.182"; \
+			echo "Requires.private: zlib"; \
+			echo "Libs.private: -lbz2"; \
+		) > $$prefix/lib/pkgconfig/libdw.pc \
```

Comment:
Hmm, `libdw.pc` is unfortunately a library that belongs to elfutils, whereas this one should be `libdwarf.pc`. Just checked `libdwarf-dev` on Ubuntu, and it doesn't contain a .pc file. So this won't fit with what distros do anyway, so I wonder if this part actually helps. Did you encounter any distros where the dev package contains `libdwarf.pc`?

Reply:
Yeah I mentioned this in the other PR. No, I haven't found a distro with `libdwarf.pc` so this isn't going to work...

We have 2 options as I see it: we can just hardcode the `--extra-library` fields for `g-ir-scanner` (simplest but hackiest), or we can generate a `libdwarf.pc` file and ALWAYS use the SDK's `libdwarf` and `libelf`, even for shared GLib builds?

Reply:
Totally agree. I think we should go with the former, so that users who insistent can build frida-gum with their distro's libdwarf package.